ui/src/main/java/com/doordeck/sdk/ui/unlock/UnlockActivity.kt

```diff
@@ -257,10 +278,20 @@ internal class UnlockActivity : BaseActivity(), UnlockView {
         }
 
         private const val TILE_ID = "tile_id"
+        private const val DEVICE = "DEVICE"
```

Comment:
Don't really like mixing 'tile_id' with 'DEVICE', we should pick one case style and stick to it